simple-example/src/main.py

```diff
@@ -14,15 +15,26 @@ def main():
 
     print("Starting example model")
 
-    # Read value from environment variable using the default value
-    # if it is not specified and convert the value to an integer
-    sequence_length = int(os.getenv("SEQUENCE_LENGTH", SEQUENCE_LENGTH_DEFAULT))
-    sequence_f0 = int(os.getenv("SEQUENCE_F0", SEQUENCE_F0_DEFAULT))
-    sequence_f1 = int(os.getenv("SEQUENCE_F1", SEQUENCE_F1_DEFAULT))
-
-    if sequence_length < 5:
-        raise Exception("Sequence_length is below the minimum value")
-
+    # Read values from environment variables but use default values if they don't exist
+    sequence_length = os.getenv("SEQUENCE_LENGTH", SEQUENCE_LENGTH_DEFAULT)
+    sequence_f0 = os.getenv("SEQUENCE_F0", SEQUENCE_F0_DEFAULT)
+    sequence_f1 = os.getenv("SEQUENCE_F1", SEQUENCE_F1_DEFAULT)
+    
+    # Check each of the values are in an acceptable format for this model
+    try:
+        sequence_length = int( sequence_length )
```

Comment:
Worth removing the spaces around the argument as it doesn't match the rest of the file?

Reply:
Do we need to run these files through Black? I guess that would get rid of the spaces around the variable anyway?

Reply:
That sounds like a good idea! Make easy to maintain consistent code style across all examples too which is good

Reply:
I like spaces.